bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx127-na.html

```diff
@@ -0,0 +1,112 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_127_na') }}
+{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-lg">
+    <h1 class="wnp-main-title">Join the community</h1>
+
+    {{ picture(
+      url='img/firefox/whatsnew/whatsnew127-na/join-community.png',
+      optional_attributes={
+        'class': 'wnp-main-image',
+        'width': '554',
+        'height': '300'
+      }
+    ) }}
+
+    <p class="wnp-main-cta">
+      <a class="mzp-c-button mzp-t-product" href="https://connect.mozilla.org" data-cta-type="button" data-cta-text="Join the community" target="_blank" rel="noopener">
```

Comment:
We should add UTM params to this external link:
```suggestion
      <a class="mzp-c-button mzp-t-product" href="https://connect.mozilla.org/?{{ utm_params }}" data-cta-type="button" data-cta-text="Join the community" target="_blank" rel="noopener">
```